app/src/main/java/com/example/areumdap/Network/AuthRepository.kt

```
@@ -126,6 +126,30 @@ object AuthRepository {
+            if (response.isSuccessful) {
+                // 성공 시에도 로컬 데이터 클리어
+                TokenManager.clearAll()
+                Result.success(Unit)
+            } else {
+                val errorMessage = when (response.code()) {
+                    404 -> "유저가 존재하지 않습니다."
+                    else -> "탈퇴 처리에 실패했습니다. (코드: ${response.code()})"
+                }
+                Result.failure(Exception(errorMessage))
+            }
+        } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)